node/heartbeat/realTime.go

```diff
@@ -0,0 +1,17 @@
+package heartbeat
+
+import "time"
+
+type RealTimer struct {
```

Comment:
Add comments for the struct and for the exported methods.

Reply:
fixed